Header.c

```diff
@@ -149,24 +195,36 @@ void Header_reinit(Header* this) {
    }
 }
 
+static const int formatWidths[][5] = {
```

Comment:
I think this should go in the Header.h file

- can be reused elsewhere
- proximity to typedef enum HeaderFormat_  {..} (which should be moved from Settings.h)